scripts/parse.py

```diff
@@ -140,8 +144,26 @@ def add_extra_metadata(dat, config):
                 d[k] = el[k]
 
 
+def add_categories(dat, config):
+    for d in dat.values():
+        name = d.get("name")
+        if name in config.categories["devtool"]:
+            d["type"] = "tool"
+        elif name in config.categories["research"]:
+            d["type"] = "research"
+        else:
+            d["type"] = "unknown"
+
+
+def add_emojis(dat):
+    for d in dat.values():
+        if d.get("description_github", None) is not None:
+            d["description_github"] = emoji_data_python.replace_colons(
+                d["description_github"])
+
+
 def write_repos(dat, config):
-    dest = os.path.join(config.path, "repos.json")
+    dest = os.path.join(config.path, "../data/" "repos.json")
```

Comment:
seemed sensible to just write this into the data dir directly, since that's where it will get used